ibis/backends/base/sql/registry/window.py

```diff
@@ -1,34 +1,19 @@
 from __future__ import annotations
 
-from operator import add, mul, sub
-
-import ibis
 import ibis.common.exceptions as com
 import ibis.expr.analysis as an
 import ibis.expr.datatypes as dt
 import ibis.expr.operations as ops
-import ibis.expr.types as ir
 
 _map_interval_to_microseconds = {
+    'Y': 31536000000000,
```

Comment:
Do we have a use case for this yet?

Reply:
Not really, going to remove it to avoid issues with leap years.

Reply:
Removed.